fedora-toolbox

```diff
@@ -246,6 +250,49 @@ enter()
 )
 
 
+list()
+(
+    to_print=""
+    if [ $1 = 1 ]; then
+        to_print="$to_print${LBC}Images created by fedora-toolbox${NC}\n"
+        output=$($prefix_sudo podman images \
+                --filter "label=com.redhat.component=fedora-toolbox")
+                #--format "table {{.ID}} {{.Repository}} {{.Tag}} {{.Created}}")
```

Comment:
`podman image` is strange in that it doesn't put the name and the tag together separated by a colon. Every time I try to copy-paste the name of an image, I need to separately select the tag, which gets just a little bit annoying after some time. :)

It's also weird that it calls it "repository", not "name".

So, to have ultimate control over our table formatting we could do something like this instead:
```
podman images --filter "label=com.redhat.component=fedora-toolbox" --format "{{.ID}} {{.Repository}}:{{.Tag}} {{.Created}}" | sed 's/ \{2,\}/\t/g' | column --separator $'\t' --table --table-columns "IMAGE ID,IMAGE NAME,CREATED"
```

We need to closely look at what podman is doing to get the column separation right, but maybe it could work?

Reply:
This does seem better than just the default output by podman, since for example podman rmi takes image name as an argument. I added this change, but leaving default output for containers until the fix for [podman#2221](https://github.com/containers/libpod/issues/2221) gets released.

Reply:
Thanks!

We might just be able to avoid using `table` at all for `podman ps`, if use `column` to create our own table.